packages/pg/lib/client.js

```diff
@@ -33,12 +33,11 @@ var Client = function (config) {
 
   // "hiding" the password so it doesn't show up in stack traces
   // or if the client is console.logged
-  const password = this.connectionParameters.password
   Object.defineProperty(this, 'password', {
+    configurable: true,
```

Comment:
i didn't really understand the advantage or diff between configurable or not 🤷‍♂

Reply:
Configurable is pretty much always the right option. It lets people monkeypatch, hook, freeze, and do all the other JavaScripty stuff when they want to, and it’s the default for most things built into the language.

Except `Object.defineProperty`. 🙃